### ISSUE TITLE
Interstitial fibrosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5059]
Definition:The interstitium is expanded by the presence of collagen that stain blue on trichrome. Tubular are not back to back, but rather separated by fibrosis and can be atrophic
Modified definition: A progressive detrimental connective tissue deposition (fibrosis) on the kidney parenchyma involving the tubules and interstitial tissue of the kidney.

**Comment by avizrosenberg** [comment]
Would it be appropriate to make this more specific: "Renal Interstitial Fibrosis"

**Comment by pnrobinson** [comment]
Currently, ,there is a term for Tubulointerstitial fibrosis
https://hpo.jax.org/app/browse/term/HP:0005576
However, the renal interstitium is defined as the intertubular, extraglomerular, extravascular space of the kidney ([source](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4594057/)).

@avizrosenberg I think we may want to re-organize the hierarchy surrounding this term. Does this make sense?
Abnormal renal interstitial morphology
--Abnormal intertubular interstitial morphology (Question -- is this synonmymous with Tubulointerstitial?)
--Abnormal extraglomerular interstitial morphology 
--Abnormal renal extravascular interstitial morphology

**Comment by nicolevasilevsky** [comment]
we can use this pattern:
https://github.com/obophenotype/upheno/blob/master/src/patterns/dosdp-dev/fibroticAnatomicalEntity.yaml

UBERON_0005215 'kidney interstitium'

**Comment by pnrobinson** [comment]
This might be good to reference for Abn interstitial morphology https://cjasn.asnjournals.org/content/10/10/1831

**Comment by avizrosenberg** [comment]
@pnrobinson I need to give this one some more consideration

**Comment by pnrobinson** [comment]
@avizrosenberg  -- thoughts?

**Comment by avizrosenberg** [comment]
> @avizrosenberg I think we may want to re-organize the hierarchy surrounding this term. Does this make sense?
> Abnormal renal interstitial morphology
> --Abnormal intertubular interstitial morphology (Question -- is this synonmymous with Tubulointerstitial?)
> --Abnormal extraglomerular interstitial morphology
> --Abnormal renal extravascular interstitial morphology

Tubulointerstitial is intended to describe processes that involve interstitium and tubules. Intertubular/extraglomerular and extravascular to a degree describe the same compartment

**Comment by avizrosenberg** [comment]
Modification of Modified definition: A progressive detrimental connective deposition of extracellular matrix, primarily fibrillar collagen, in the interstitial renal compartment. Associated tubules can be atrophic.

**Comment by pnrobinson** [comment]
@avizrosenberg 
So my question is really whether fibrosis "always" affects both the interstitium and the tubuli (in this case, we can use the existing term for Tubulointerstitial fibrosis: https://hpo.jax.org/app/browse/term/HP:0005576). If this is not the case then we would have


Tubular fibrosis
Interstitual fibrosis
Tubulointerstitial fibrosis


Currently, the Tubulointerstitial fibrosis has as a synonym "Renal interstitial fibrosis".

So, should we just stay with this term?

**Comment by pnrobinson** [comment]
@avizrosenberg  -- I have slightly modified the definition as follows. I will add a statement to 
"Tubulointerstitial fibrosis" that the term should be avoided for new annotations. Please reopen if needed!

Definition: The accumulation of collagen and related extracellular matrix (ECM) molecules in the interstitium of the kidney. The interstitium is expanded by the presence of collagen that stain blue on trichrome. Tubules are not back to back, but rather separated by fibrosis and can be atrophic. [PMID:22449945]

comment:  Tubular atrophy (TA) is defined as loss of specialized transport and metabolic capacity and typically manifested by small tubules with cells with pale cytoplasm or dilated, thin tubules. TA is usually associated with IF, but probably has distinct mechanisms related to blood flow, glomerular filtration rate (GFR) or tubular continuity loss. However, IF and TA are separable, as shown by the profound TA in renal artery stenosis, which characteristically has little or no fibrosis (or inflammation).
